### ISSUE TITLE
(Feature request) Spacing in Tree

### ISSUE THREAD

**Issue by eeshaa1** [TriliumNext/Trilium#4256]
### Describe feature

A way to add a totally blank "entry" in the tree would be nice, like how you can add a 'Spacing' section when configuring the launchbar.

**Comment by sottey** [comment]
Couple notes on meichthys response:

1) I wasn't able to find any broken icons, but if you just select any icon, then select Owned Attributes and change the value of the #iconClass property to something invalid. This results in no icon. Removing the default title will give you a blank line in the tree
2) If you do this, know that it seems to break the drag and drop of that blank note. So if you want it in a specific location, create it where you want it as you won't be able to move it.

**Comment by eeshaa1** [comment]
Thanks everyone

**Comment by zadam** [comment]
Hmm, it's actually kinda weird that drag & drop is broken in this case.

Moving by CTRL-Up and CTRL-Down still works.

**Comment by sottey** [comment]
Nah, I am wrong. That Fancy Tree console log shows on first drag after reload for any node. weird.

**Comment by eliandoran** [comment]
There's actually a way to get an empty icon that doesn't cause any issues:

```
#iconClass="bx bx-empty"
```
